packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -534,7 +595,7 @@ export type UmbrellaStoreState<M extends BaseMetadata> = {
    * Versions by roomId
    * e.g. { 'room-abc': {versions: "all versions"}}
    */
-  versionsByRoomId: Record<string, HistoryVersion[]>;
+  versionsByRoomId: Record<string, Record<string, HistoryVersion>>;
```

Comment:
Why did you change this data structure here internally? When we fetch these from the backend, it's a list, and when we expose it from the hook it's also a list. In between we now have to convert the list to a Map, and back to an array again when we read it.

Is this done to remove "duplicate" version IDs? Ideally there would not be such duplicates in the first place, right?